mobile/src/assets/icons.ts

```diff
@@ -20,7 +20,8 @@ export const transactionIcons = {
   done: require('./icons/check-circle.png'),
   waiting: require('./icons/clock.png'),
   camera: require('./icons/camera.png'),
-  copy: require('./icons/content-copy.png')
+  copy: require('./icons/content-copy.png'),
```

Comment:
Oh, I see that you don't use font/vector-icons...
https://github.com/oblador/react-native-vector-icons could be interesting.